SimG4Core/PhysicsLists/src/CMSEmNoDeltaRay.cc

```diff
@@ -154,53 +157,43 @@ void CMSEmNoDeltaRay::ConstructProcess() {
         msc_el->SetRangeFactor(0.04);
         msc->AddEmModel(0, msc_el, reg);
       }
-      pmanager->AddProcess(msc, -1, 1, 1);
-      pmanager->AddProcess(new G4hhIonisation, -1, 2, 2);
-      pmanager->AddProcess(new G4eBremsstrahlung, -1, -3, 3);
+      pmanager->AddProcess(msc, -1, 1, -1);
```

Comment:
@civanch what is the expected effect of these changes in physics lists, in the process intialization? There are several of them...

Reply:
@fabiocos , this is an old EM configuration which these days are not used. The choice was to remove it or update to be consistent with Geant4 10.4. This EM physics may be applied for some exotic studies(difficult to predict) or will not be used any more....

Reply:
@civanch thanks, my question is actually for all the changes to physics list in this PR, but I understand they fall in the same category. I will integrate this PR in next IB